backend/windmill-common/src/utils.rs

```diff
@@ -244,9 +244,12 @@ pub fn generate_lock_id(database_name: &str) -> i64 {
 pub async fn report_critical_error(error_message: String, _db: DB) -> () {
     tracing::error!("CRITICAL ERROR: {error_message}");
 
+    let mute = check_mute_status(&_db).await.unwrap_or(false);
```

Comment:
you do not want to check mute status for every alert, instead you want to have this been saved globally as a RwLock and whenever there is a change of global setting notified through the notify channel in main.rs, change it in rw_lock. Will save many unecessary transactions

Reply:
oh cool, didn't see the global_settings notify channel.

done and tested.